Hazelnut/src/EditorLayer.cpp

```diff
@@ -303,7 +303,7 @@ namespace Hazel {
 		if (e.GetRepeatCount() > 0)
 			return false;
 
-		int gizmoChange = -2;
+		bool canGizmoChange = !ImGuizmo::IsUsing();
```

Comment:
no need to store it as a variable, it is only used in the switch case anyway (once)

```suggestion

```

Reply:
yes, but in the future can, be more conditions to change gizmo

Reply:
We can change it in the future then.. now it's just bloating the code...